### DISCUSSION TITLE
Filter by Severity

### DISCUSSION THREAD

**Discussion by FaisalBudiono** [folke/trouble.nvim#454] [Q&A]
The previous version has a way to change severity on the fly by pressing <kbd>s</kbd> to filter out the severity.

In the current version we should state the severity when we call the Trouble like `Trouble diagnostics filter.severity=vim.diagnostic.severity.ERROR`

Am I missing the somewhere to read to change this setting?

**Comment by folke** [comment]
Just added that back
